scripts/run_integration_tests_rate_limiter.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
GIven you know the `rate_limiter` tests are gonna run on the metrics machine right after the performances tests, you should not create a new script (we already have too many of them...).

Reply:
Yes I agree we have too many scripts. But I think it's a bad idea to overload the meaning of an existing script. Similarly why I asked to create a new top level job even if it is going to be scheduled by Jenkins onto the same machine.